components/src/core/info-list-item/info-list-item.component.scss

```diff
@@ -7,6 +7,7 @@ $denseHeight: 52px;
 
 .pxb-info-list-item {
     display: flex;
+    flex-direction: column;
```

Comment:
`display: block` will give the same result with better browser support

Reply:
You're right visually it looks correct and it's more simple, just running into a problem with the unit tests now.  

I'm gonna fix these tests then it should be good to go.